Assets/Prefabs/Enemies/brad_enemyParent.prefab

```diff
@@ -60,7 +60,7 @@ MeshRenderer:
   m_RenderingLayerMask: 1
```

Comment:
tbh I'm really confused thy this prefab was created or why its here since its really similar to the space fly adult one, either justify why it was created or remove it